xclim/utils.py

```diff
@@ -160,7 +160,7 @@ def create_ensemble(ncfiles, dim='sim'):
     return ens
 
 
-def ensemble_statistics(ens, stats={'type': 'perc', 'values': [10, 50, 90], 'time_block': []}, ):
+def ensemble_percentiles(ens, values=[10, 50, 90], time_block=[]):
```

Comment:
Using a mutable object (here a list []) as a default value is dangerous, because it is initialized once, not every time the function is called. https://docs.python-guide.org/writing/gotchas/

Reply:
noted thanks...